content/nginxaas-azure/monitoring/enable-monitoring.md

```diff
@@ -34,14 +34,26 @@ Monitoring your application's performance is crucial for maintaining its reliabi
 2. In the navigation pane under **Settings**, select the **NGINX monitoring** section.
 3. Turn on the **Send metrics to Azure Monitor** setting.
 
-## View metrics with Azure Monitor metrics explorer
+## Review metrics
+
+Once you have enabled monitoring, you can view the metrics using the Azure portal or the Azure Monitor API.
+
+{{< note >}} NGINX Agent periodically gathers connection and request statistics using an internal HTTP request. An Azure service health probe checks for status using a TCP connection for each listen port in the NGINX configuration, incrementing the connection count for each port.
+
+This contributes to minimal traffic and should not affect these metrics significantly, but you might see an unexpected ammount of connections and requests.{{</note>}}
+
+{{< important >}}If some of your deployment's metrics are intermittently missing in Azure monitor, it may indicate that the deployment's underlying compute resources are being exhausted.
```

Comment:
```suggestion
{{< important >}}If some of your deployment's metrics are intermittently missing in Azure monitor, it may indicate that the underlying resources for your deployment are being exhausted.
```